guides/common/modules/ref_email-notification-types.adoc

```diff
@@ -1,16 +1,16 @@
-[id="Notification_Types_{context}"]
-= Notification Types
+[id="Email_Notification_Types_{context}"]
+= Email Notification Types
 
-The following are the notifications created by {Project}:
+Events for which you can receive email notifications from {Project} include the following:
 
 * *Audit summary*: A summary of all activity audited by {ProjectServer}.
 * *Host built*: A notification sent when a host is built.
 * *Host errata advisory*: A summary of applicable and installable errata for hosts managed by the user.
-* *OpenSCAP policy summary*: A summary of OpenSCAP policy reports and their results.
+* *Compliance policy summary*: A summary of OpenSCAP policy reports and their results.
 * *Promote errata*: A notification sent only after a Content View promotion.
 It contains a summary of errata applicable and installable to hosts registered to the promoted Content View.
 This allows a user to monitor what updates have been applied to which hosts.
-* *Puppet error state*: A notification sent after a host reports an error related to Puppet.
-* *Puppet summary*: A summary of Puppet reports.
 * *Sync errata*: A notification sent only after synchronizing a repository.
```

Comment:
This lists notifications rather than events, for example **\*Compliance policy summary\*: A summary of OpenSCAP policy reports and their results** is not an event. So I think the introductory sentence should stay as it was or be reworded in a way that still introduces items in the list as notifications.